wire-ios-mono.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -1,6 +1,15 @@
 {
   "object": {
     "pins": [
+      {
+        "package": "DifferenceKit",
+        "repositoryURL": "https://github.com/wireapp/DifferenceKit",
+        "state": {
+          "branch": null,
+          "revision": "eaf29c583df322d1d7c4c72c46a59eeb33bb54bf",
```

Comment:
It uses
https://github.com/wireapp/DifferenceKit/releases/tag/1.3.0
instead of
https://github.com/wireapp/DifferenceKit/releases/tag/1.3.0_xcframework_14.3.1

Could be forced to use commit
f9e042ddefdbf045d394ff211f1c247f1d2e488a

Reply:
I think it's fine